TestPlaybooks/playbook-GoogleVault-test.yml

```diff
@@ -235,85 +201,19 @@ tasks:
     view: |-
       {
         "position": {
-          "x": 265,
-          "y": 195
-        }
-      }
-    note: false
-    timertriggers: []
-    ignoreworker: false
-  "13":
-    id: "13"
-    taskid: d82bf06f-3680-4c34-8605-73729b6f8728
-    type: regular
-    task:
-      id: d82bf06f-3680-4c34-8605-73729b6f8728
-      version: -1
-      name: Delete Context
-      description: Deleting context so that we can get an updated list of matters
-        later
-      scriptName: DeleteContext
-      type: regular
-      iscommand: false
-      brand: ""
-    nexttasks:
-      '#none#':
-      - "14"
-      - "24"
-    scriptarguments:
-      all:
-        simple: "yes"
-      index: {}
-      key: {}
-      keysToKeep: {}
-      subplaybook: {}
-    separatecontext: false
-    view: |-
-      {
-        "position": {
-          "x": 265,
-          "y": 1085
-        }
-      }
-    note: false
-    timertriggers: []
-    ignoreworker: false
-  "14":
-    id: "14"
-    taskid: 40a9a9b8-043e-4b60-8a2e-95947b9f8030
-    type: regular
-    task:
-      id: 40a9a9b8-043e-4b60-8a2e-95947b9f8030
-      version: -1
-      name: Get "First matter" Matter
```

Comment:
why did we remove that one?

Reply:
This test is quite old so it wasn't so optimal. For example, it had hardcoded IDs that are no longer exist making it fail. I had to change several flows. :/